helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -217,10 +217,16 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "podLabels": {
```

Comment:
Please keep `podLabels` for now and mark it as deprecated similar to how it's done here https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/templates/NOTES.txt#L24

Reply:
@dmitryax do you want me to add back the old implementation too?

or just leave the schema?